### PR TITLE
Added a camel-bindy extension #653

### PR DESCRIPTION
**Pull request by aldettinger**

### CODE REVIEW

File: .github/workflows/pr-build.yaml [hunk 1]
```diff
@@ -347,6 +347,7 @@ jobs:
             -Dnative \
             -Ddocker \
             -DskipTests \
+            -pl :camel-quarkus-integration-test-bindy \
```
Comment (aldettinger) [inline .github/workflows/pr-build.yaml]: Allocated in this pr build slot by default.

**Comment by aldettinger** [comment]
ok to test

**Comment by oscerd** [comment]
There is no need to run the jenkins job, I believe. @lburgazzoli ?

**Comment by aldettinger** [comment]
Yeah, I just realized this, as the pr-builds are ok. I will just change the category from saas to misc and push on master.

**Comment by aldettinger** [comment]
Ok closing, I tried to stopped the unneeded build. Not sure it will stop.

**Pull request closed without merging**